build-utils/compare-versions.js

```diff
@@ -0,0 +1,21 @@
+module.exports = function compareVersions(a, b) {
```

Comment:
i wonder if this would be simpler if we used one of `semver`'s comparison functions (altho it wouldn't handle `tp`)

Reply:
I would prefer to keep these files dependencies-free if possible, to make it easier to re-use them by simply cloning this repository.

Reply:
I'm fine with that in this case, but I think that's flat out a bad goal to have in general. If we want this stuff to be usable by outside tools we should publish it as a separate package :-/

Reply:
Yeah, "zero deps" shouldn't be a goal. However, in this case it was easy enough to keep it with the custom implementation.